pgcli/config.py

```diff
@@ -1,23 +1,17 @@
 import shutil
 from os.path import expanduser, exists
-try:
-    from ConfigParser import SafeConfigParser as ConfigParser
-except ImportError:
-    from configparser import ConfigParser
+from configobj import ConfigObj
 # from prompt_toolkit.contrib.pdb import set_trace
 
 
 def load_config(filename, default_filename=None):
     filename = expanduser(filename)
-    parser = ConfigParser()
+    config = ConfigObj(filename, interpolation=False)
 
-    # parser.read will not fail in case of IOError,
-    # so let's not try/except here.
     if default_filename:
-        parser.read(default_filename)
+        config.merge(load_config(default_filename))
```

Comment:
This has changed the order of precedence. The original code would read the default config then apply the overrides from the user config on top. 

But this change here makes it the other way around. So the default settings will always take precedence which is not a desirable behavior.